test/WorkOSTests/Services/AuditLogs/AuditLogsSeviceTest.cs

```diff
@@ -0,0 +1,153 @@
+﻿namespace WorkOSTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Net.Http;
+    using Newtonsoft.Json;
+    using WorkOS;
+    using Xunit;
+
+    public class AuditLogsSeviceTest
+    {
+        private readonly HttpMock httpMock;
+
+        private readonly AuditLogsService service;
+
+        private readonly AuditLogEvent auditLogEventPayload;
+
+        public AuditLogsSeviceTest()
+        {
+            this.httpMock = new HttpMock();
+            var client = new WorkOSClient(
+                new WorkOSOptions
+                {
+                    ApiKey = "test",
+                    HttpClient = this.httpMock.HttpClient,
+                });
+            this.service = new AuditLogsService(client);
+
+            this.auditLogEventPayload = new AuditLogEvent
+            {
+                Action = "document.updated",
+                OccurredAt = DateTime.Now,
+                Version = 1,
+                Actor = new AuditLogEventActor
+                {
+                  Id = "user_123",
```

Comment:
nit: Looks like some of these fields are indented with 2 spaces instead of 4. Could we standardize on 4?

Reply:
Updating. Is there a standard formatter we can add to this repo? Not familiar with the C#/.Net ecosystem.

Reply:
When I was leaving the .NET ecosystem `dotnet format` was just taking off: https://github.com/dotnet/format

I haven't used it, but I think that would be the one we should investigate.